llvm/test/Feature/Repo/Inputs/remove_GO_ticketnode.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import re
+
+def remove_function_ticketnode (functions, inputfile, outputfile):
+  """
+  Remove the repo_ticket metadata of the `functions` from `file`.
+  """
+  define = "@{0}()"
+  functions = [define.format(i) for i in functions]
+  functions = tuple(functions)
+  with open(inputfile, 'r') as f:
+    lines = f.readlines()
+  with open(outputfile, 'w') as f:
+    for line in lines:
+      if line.startswith("define") and any(function in line for function in functions):
+        line = re.sub('!repo_ticket ![0-9]', '', line)
+      f.write(line)
+
+def remove_variable_ticketnode (variables, inputfile, outputfile):
+  """
+  Remove the repo_ticket metadata of the `variables` from `file`.
+  """
+  define = "@{0} ="
+  variables = [define.format(i) for i in variables]
+  variables = tuple(variables)
+  with open(inputfile, 'r') as f:
+    lines = f.readlines()
+  with open(outputfile, 'w') as f:
+    for line in lines:
+        if line.startswith(variables):
+          line = re.sub(', !repo_ticket ![0-9]', '', line)
+        f.write(line)
+
+def main():
+  parser = argparse.ArgumentParser()
+  group = parser.add_mutually_exclusive_group()
+  group.add_argument('-f','--function-list', default=None, nargs='+', help='A function list whose ticketnode need to be removed.')
+  group.add_argument('-v','--variable-list', default=None, nargs='+', help='A global variable list whose ticketnode need to be removed.')
+  parser.add_argument('-i','--input-file', default=None, help='the input file name.')
+  parser.add_argument('-o','--output-file', default=None, help='the output file name.')
+  args = parser.parse_args()
+
+  if args.function_list != None:
```

Comment:
After combined two functions, I need to pass argument of `isFunction` to the new function. Therefore, I don't think I can remove the condition now.

Reply:
I don’t follow. Adding an extra argument to `remove_GO_ticketnode()` doesn’t prevent `if args.function_list`… from being redundant. This can simply become:
~~~python
remove_GO_ticketnode(args.function_list, True, args.input_file, args.output_file)
remove_GO_ticketnode(args.variable_list, False, args.input_file, args.output_file)
~~~
(Where `function_list` and `variable_list` both default to `[]`.)

Reply:
I see. Fixed, thanks.